### DISCUSSION TITLE
ElectronStudio的Unity工程源码

### DISCUSSION THREAD

**Discussion by 2468318669** [peng-zhihui/ElectronBot#223] [Q&A]
新手，自己做了个机械手想替换掉小人来控制机械手的移动，目前没有做到，想知道代码里控制小人移动的方法，最好能有大佬大概讲一下方法的作用，万分感谢

**Comment by Kitem** [comment]
您好，我是焕挺，您的邮件已经收到，我会尽快回复！

**Reply by DaoThuc** [reply]
我想索要ElectronBot Studio Unity的开源代码，需要稍微改变一下界面。

**Comment by VendettaMask** [comment]
来信已收，谢谢！

**Comment by BLDC888** [comment]
您好：你的来信我已收到，我会尽快处理的。

**Comment by Kitem** [comment]
您好，我是焕挺，您的邮件已经收到，我会尽快回复！

**Comment by DaoThuc** [comment]
我想索要ElectronBot Studio Unity的开源代码，需要稍微改变一下界面。

**Comment by DaoThuc** [comment]
谢谢

Vào CN, 17 thg 11, 2024 lúc 09:57 Kitem ***@***.***> đã viết:

> 您好，我是焕挺，您的邮件已经收到，我会尽快回复！
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/peng-zhihui/ElectronBot/discussions/223#discussioncomment-11281298>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWKG6SA65SXHLWRZTFORV2L2BAAYDAVCNFSM6AAAAABR5OFNU2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMRYGEZDSOA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***
> com>
>
-- 

*Trân trọng,*

--------------

*Đào Văn Thức*

Electrical Engineer


*Address*: Bac 1 hamlet, Thuy Duong commune, Thuy Nguyen district, Hai
Phong City, Vietnam

*Tel:* 0356 224 796

Email: ***@***.***

***********************
